src/modules/sdk/objections/service.ts

```diff
@@ -0,0 +1,38 @@
+import { AxiosRequestConfig } from "axios";
+import logger from "../../../utils/logger";
+import { INTERNAL_API_URL } from "../../../utils/properties";
+import { getAxiosRequestConfig, HTTP_POST, makeAPICall } from "./axios.client";
+import { ObjectionPatch } from "./types";
+
+/**
+ * Create a new objection for the given company.
+ *
+ * @param {string} companyNumber the company number
+ * @param {string} token the bearer security token to use to call the api
+ *
+ * @returns {string} the id of the newly created objection
+ * @throws {ApiError}
+ */
+export const createNewObjection = async (companyNumber: string, token: string): Promise<string> => {
+  logger.info(`Creating a new objection for company number ${companyNumber}`);
+
+  const createNewObjectionUrl = `${INTERNAL_API_URL}/company/${companyNumber}/strike-off-objections`;
```

Comment:
Guessing this URL string might become a constant when we add more calls to the API end-points, but not needed yet.

Reply:
it will be re-used so made into into a constant/function